scrapers_next/pr/committees.py

```diff
@@ -0,0 +1,95 @@
+from spatula import HtmlPage, HtmlListPage, CSS, SelectorError, URL
+from openstates.models import ScrapeCommittee
+import re
+
+
+class SenateCommitteeDetail(HtmlPage):
+    example_source = (
+        "https://senado.pr.gov/comisiones/comisión-de-asuntos-de-vida-y-familia"
+    )
+
+    def process_page(self):
+        com = self.input
+        com.add_source(self.source.url, "Committee Detail Page")
+
+        members = CSS("div.panel-body div.senator_cont a").match(self.root)
+
+        for member in members:
+            name = CSS("span.name").match_one(member).text_content()
+            role = CSS("span.position").match_one(member).text_content()
+            if role == "":
```

Comment:
`if not role` is a bit broader. Is there a reason we're doing `if role == ""`?

Reply:
Changed this and restructured the try statements in most recent commit